__tests__/decorators.spec.ts

```diff
@@ -66,4 +66,110 @@ describe('Decorators', () => {
       expect(consoleSpy).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('arraySort', function() {
+    
+    it('should sort testing array by name property in ascending order', function() {
+      class TestArraySortAscending {
+        @arraySort('name')
+        testingArray = [
+          {
+            name: 'b'
+          },
+          {
+            name: 'a'
+          },
+          {
+            name: 'c'
+          }
+        ]; 
+      }
+      
+      const instance = new TestArraySortAscending();
+      
+      expect(instance.testingArray).toEqual([
+        {
+          name: 'a'
+        },
+        {
+          name: 'b'
+        },
+        {
+          name: 'c'
+        }
+      ]);
+    });
+
+    it('should sort testing array by name property in descending order', function() {
+      class TestArraySortDescending {
+        @arraySort('name', true)
+        testingArray = [
+          {
+            name: 'b'
+          },
+          {
+            name: 'a'
+          },
+          {
+            name: 'c'
+          }
+        ]; 
+      }
+      const instance = new TestArraySortDescending();
+      expect(instance.testingArray).toEqual([
+        {
+          name: 'c'
+        },
+        {
+          name: 'b'
+        },
+        {
+          name: 'a'
+        }
+      ]);
+    });
+
+    it('should throw error if value is not an array', function() {
+      class TestTypeError {
+        @arraySort('name', true)
```

Comment:
Few improvements: 
1. Can we change it to `SortBy`?
2. Can we support types? `SortBy('name', { type: number/date/string })`
3. Can we support non-object items? `e.g: [3, 4, 6]`

Reply:
Hi @NetanelBasal ,

All three improvements have been addressed. Feel free to leave more feedback :)

Yuchao